docs/content/doc/advanced/customizing-gitea.en-us.md

```diff
@@ -72,6 +72,11 @@ matching source.
 Any statement contained inside `{{` and `}}` are Gitea's template syntax and
 shouldn't be touched without fully understanding these components.
 
+### Customizing startpage / homepage
+
+Copy [`home.tmpl`](https://github.com/go-gitea/gitea/blob/master/templates/home.tmpl) for your version of Gitea from `templates` to `custom/templates`.
+Edit as you wish.
+
```

Comment:
Is that can be need to provide information about start/restart service/docker image ... ?

Reply:
@jaqra 
```diff
diff --git a/docs/content/doc/advanced/customizing-gitea.en-us.md b/docs/content/doc/advanced/customizing-gitea.en-us.md
index 33e6ea8ee..362f4fb25 100644
--- a/docs/content/doc/advanced/customizing-gitea.en-us.md
+++ b/docs/content/doc/advanced/customizing-gitea.en-us.md
@@ -76,6 +76,7 @@ shouldn't be touched without fully understanding these components.
 
 Copy [`home.tmpl`](https://github.com/go-gitea/gitea/blob/master/templates/home.tmpl) for your version of Gitea from `templates` to `custom/templates`.
 Edit as you wish.
+Dont forget to restart your gitea to apply the changes.
 
 ### Adding links and tabs
```

Reply:
Good to me. Make no sense how it is :) but others.